docs/guides/admin/docs/installation/ansible.md

```diff
@@ -0,0 +1,111 @@
+Install via Script using the Repository
+===========================================================================
+
+Installing Opencast can be complex, so we have built a set of Ansible scripts to handle the installation and 
+configuration.  These scripts should handle most common installation setups, and are ideal for someone wishing to 
+quickly and easily set up Opencast for testing.  These scripts are also a good base upon which to build your own, local
+configuration management scripts.
+
+The Ansible scripts are tested on long term stable releases of Debian, Ubuntu, and CentOS.  Currently this is Debian 8,
+Ubuntu 16.04, and 18.04, and CentOS 7.  Other releases, and distributions should work, but are untested.
+
+
+Setup
+-----
+
+These scripts use [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html), which
+allows you to easily and automatically manage large numbers of machines via SSH.  Install Ansible before continuing
+with this guide.
+
+The scripts can be found [here](https://github.com/opencast/oc-config-management).  If you are familiar with `git`
```

Comment:
If we merge this now this link is broken.

Reply:
The intention is to fork the code at https://github.com/gregorydlogan/oc-config-management at merge time :)

Reply:
I was thinking this is the first ever case of the documentation been present before the code!